Gopkg.toml

```diff
@@ -0,0 +1,34 @@
+# Gopkg.toml example
+#
+# Refer to https://golang.github.io/dep/docs/Gopkg.toml.html
+# for detailed Gopkg.toml documentation.
+#
+# required = ["github.com/user/thing/cmd/thing"]
+# ignored = ["github.com/user/project/pkgX", "bitbucket.org/user/project/pkgA/pkgY"]
+#
+# [[constraint]]
+#   name = "github.com/user/project"
+#   version = "1.0.0"
+#
+# [[constraint]]
+#   name = "github.com/user/project2"
+#   branch = "dev"
+#   source = "github.com/myfork/project2"
+#
+# [[override]]
+#   name = "github.com/x/y"
+#   version = "2.4.0"
+#
+# [prune]
+#   non-go = false
+#   go-tests = true
+#   unused-packages = true
+
+
+[[constraint]]
+  name = "github.com/hashicorp/vault"
+  version = "0.10.3"
```

Comment:
In case you didn't know (like I didn't), this defaults to `^`, which I think is what we want for Vault: https://golang.github.io/dep/docs/Gopkg.toml.html#version

Reply:
This is what `dep init` generated for me.

Reply:
I just meant that it means `>= 0.10.3, < 0.11` effectively which seems right because Vault can have pretty major changes between versions 0.x and 0.(x+1)